amaznot/resources/views/auth/logout.blade.php

```diff
@@ -0,0 +1,13 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class = "flex justify-center">
+        <div class = "w-4/12 bg white p-6 rounded-lg">
+            <form action = "{{ route('logout')}}" method = "post">
+
+            </form>
```

Comment:
Do we need a dedicated page for logging out? Isn't that normally handled by a button?

Reply:
I thought since @fullKTRL was working on the front end portion, I would make the blade file so he would know which to do the front-end part. Anyways, I'll try to make a simple form for this and @fullKTRL can make it look nice after